packages/headless/src/controllers/tab/headless-tab.ts

```diff
@@ -27,6 +28,11 @@ export interface TabOptions {
    * `@objecttype==Message`
    */
   expression: string;
+
+  /**
+   * A unique identifier for the tab. The value will used as the originLevel2 when the tab is active.
+   */
+  id?: string;
```

Comment:
What do you think of `id` as the option name compared to `name` or other?

Reply:
I think `id` is more fitting for tabs rather than `name`

Reply:
I also think `id` is more fitting, but I think we should give an example in the description of what that could look like.

Usually, when a framework asks me for an `id` and doesn't provide examples, I have to go looking for whether there's validation on it (e.g., no spaces? no dashes? no special characters?).

Perhaps we should also include a link to some doc explaining what originLevel2 is.